tests/test_radar_dataset.py

```diff
@@ -50,11 +54,39 @@ def test_radar_dataset(tmp_path):
         target,
         max_distance=max_distance,
         min_neighbours=min_neighbours,
+        max_edge_distance=max_edge_distance,
     )
 
+    # Tests that filtering to have zero graphs throws an error
     with pytest.raises(ValueError) as excinfo:
         RadarDataset(tmp_path, features, target, max_distance=0.0)
     assert "No graphs" in str(excinfo.value)
     with pytest.raises(ValueError) as excinfo:
         RadarDataset(tmp_path, features, target, min_neighbours=10**8)
     assert "No graphs" in str(excinfo.value)
+
+    # Tests that if the maximum edge distance is too small, then only self-loops are found
+    dataset = RadarDataset(
```

Comment:
Very nice addition. RadarDataset is created a few times however. Perhaps better use fixture for this as described in https://github.com/point-cloud-radar/bird-cloud-gnn/issues/51?

Reply:
This is testing `RadarDataset` functionalities. Fixtures won't help.